account/account.go

```diff
@@ -200,7 +198,7 @@ func (account *Account) TransactionHashDeployAccount(tx rpc.DeployAccountType, c
 			txn.MaxFee,
 			account.ChainId,
 			[]*felt.Felt{txn.Nonce},
-		)
+		), nil
```

Comment:
nit: would you mind removing the calldata array in the roc.DeployAccountTxnV3 calculation, it doesn't look like it's used.

calldata := []*felt.Felt{txn.ClassHash, txn.ContractAddressSalt}
calldata = append(calldata, txn.ConstructorCalldata...) //nolint:all